### DISCUSSION TITLE
Size of array element of type 'blake2b_state' (aka 'struct crypto_generichash_blake2b_state') (361 bytes) isn't a multiple of its alignment (64 bytes)

### DISCUSSION THREAD

**Discussion by Luck-666** [jedisct1/libsodium#1424] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/dca91495-6bc7-45c7-a283-e1d8b7c743b6)
Xcode 16.1 libsodium pod isntall version 1.0.12,can't build success

**Comment by jedisct1** [comment, marked as answer]
That was fixed in 2017.

Current version is 1.0.20-stable.
